cmd/convox/install.go

```diff
@@ -111,6 +111,10 @@ func init() {
 				Usage: "internet gateway id to use in existing vpc",
 			},
 			cli.BoolFlag{
+				Name:  "no-autoscale",
```

Comment:
Nice, I wasn't aware of `BoolTFlag`. Good to know but for what it's worth, after looking it up, later version of`urfave/cli` remove that type and gave `BoolFlag` a default value.